cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -343,7 +354,7 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
           {{- range $i, $e := .Frontend.Backends}}
           {
             name: '{{upper .Name}}_BASE_URL'
-            value: 'https://{{.Name}}.internal.${containerAppsEnvironment.outputs.defaultDomain}'
+            value: 'https://{{.Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
```

Comment:
The front end can not access the URL with `internal.` inside. Delete `internal.` will make it work well.

Reply:
That is interesting. Can you check with `http://{{.Name}}` works as well? `internal` was in their previous documentation, it looks like it did end up changing based on these new [docs](https://learn.microsoft.com/en-us/azure/container-apps/connect-apps?tabs=bash).

I will have to play around with this a little to test this out, will get back to you in a day or two.

Reply:
I tried `http://{{.Name}}`, it doesn't work. It's not hard to understand: When the `API_BASE_URL` used in frontend application, it is used in js file in browser, it's not internal network.